tests/qmctl-test/test_qmctl.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../e2e/lib/utils
+
+info_message "Running qmctl command..."
+
+QMCTL_SCRIPT="../../tools/qmctl/qmctl"
+
+if [ ! -f "$QMCTL_SCRIPT" ]; then
+  info_message "FAIL: qmctl script not found at $QMCTL_SCRIPT"
```

Comment:
same for all of these FAIL

Reply:
It seems like, on the comment @Yarboa sent me, for example, `info_message` is used everywhere.
Should I add an issue to make it echo everywhere, or should I continue as in this file `tests/ffi/sys_read_only/test.sh`?

@Yarboa, @dougsland, what do you think?